src/Auth/Auth.js

```diff
@@ -105,6 +105,14 @@ function acquireTokens(callback) {
   return currentProvider.acquireTokens(callback);
 }
 
+function acquireTokensByRequest(tokenReq, callback) {
+  initProviderIfNull();
+  if (currentProvider === undefined) {
+    return undefined;
+  }
+  return currentProvider.acquireTokensByRequest(tokenReq, callback);
```

Comment:
The argument `callback` is not used in AuthDev nor in AuthMSAL, should we remove it ?

Reply:
done